package.json

```diff
@@ -46,6 +47,7 @@
   "devDependencies": {
     "@types/node": "^22.10.1",
     "@types/react": "^18.3.13",
+    "ajv": "^7.2.4",
```

Comment:
I don't know what these new dependencies we are are adding here. The bug we are experiencing is because the 
" npm error Missing: @testing-library/dom@10.4.0 from lock file"